app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/PhotoViewFragment.kt

```diff
@@ -56,6 +58,36 @@ class PhotoViewFragment : Fragment() {
                 })
     }
 
+    /**
+     * Set up the toolbar for editing photos
+     */
+    private fun setUpToolbar() {
+        toolbarFragment = activity.supportFragmentManager.findFragmentById(R.id.expand_photo_toolbar_fragment) as PhotoNavigatonToolbarFragment
+
+        // hide the toolbar when image is first loaded
+        activity.supportFragmentManager.beginTransaction()
+                .setCustomAnimations(R.anim.tooltip_enter, R.anim.tooltip_exit)
+                .hide(toolbarFragment)
+                .commit()
+
+        photoView?.setOnClickListener {
+            isActiveEdit = !isActiveEdit
+            if (isActiveEdit) {
```

Comment:
This is more of a design question more than anything else. I feel like this code should live outside of the PhotoViewFragment & live on its holding activity.

Here are my reasons:

1.  Your code becomes more modular. If we do it this way the fragment who is only in charge of displaying images will also know about the edit toolbar which feels wrong.
2. It will make the edit behavior more consistent with what the current android gallery does. If you put it outside it will make it feel like we are scrolling through images and the toolbar will remain on the same spot.
3. On the activity you have more information about who invoked you and the answer that you are gonna give back if a photo is deleted, edited or something happened that we need to signal the calling activity